server/tests/api.test.js

```diff
@@ -6,9 +6,29 @@ const request = supertest(app);
 describe('API Endpoints', () => {
   it('Should retrieve the game info for a game based on a valid query parameter ID', async (done) => {
     const res = await request.get('/game_carousel_info?id=1');
-
     expect(res.status).toBe(200);
-    expect(res.body.length).toBeGreaterThan(0);
+    expect(res.body.length).toEqual(1);
+    done();
+  });
+  it('Should have a status of 500 for a query parameter ID that is invalid (not 1-100)', async (done) => {
+    const res = await request.get('/game_carousel_info?id=150');
+    expect(res.status).toBe(500);
+    done();
+  });
+  it('Should contain a game_id property on a valid query parameter ID', async (done) => {
+    const queryId = 7;
+    const res = await request.get(`/game_carousel_info?id=${queryId}`);
+    expect(res.status).toBe(200);
+    expect(res.body[0]).toEqual(expect.objectContaining({
+      game_id: queryId,
+    }));
+    done();
+  });
+  it('Should contain a picture from an S3 bucket on a valid query parameter ID', async (done) => {
```

Comment:
Good job not only testing if you get the proper response but if that response contains things in the body that you expect it to send back